pycon/finaid/views.py

```diff
@@ -425,3 +425,33 @@ def get_value(name, object):
         writer.writerow(data)
 
     return response
+
+
+@login_required
+def receipt_form(request):
+    receipts = Receipt.objects.filter(user__username=request.user.username)
+
+    if request.method == 'POST':
+        form = ReceiptForm(request.POST, request.FILES)
+
+        if form.is_valid():
+            form.instance.user = request.user
+            form.instance.application = request.user.financial_aid
+            form.save()
+
+            # Display a message to user
+            messages.add_message(request, messages.INFO,
+                                _(u"Receipt submitted"))
+
+    else:
+        form = ReceiptForm()
+    return render(request, "finaid/receipt_form.html", {
+        'form': form,
+        'receipts': receipts
+    })
+
+
+@login_required
+def receipt_delete(request, pk):
+    receipt = get_object_or_404(Receipt, pk=pk).delete()
```

Comment:
The `delete` method on querysets doesn't return anything useful in Django 1.8 and earlier.